backend/routes/auth.go

```diff
@@ -0,0 +1,15 @@
+package routes
+
+import (
+	"api/controller"
+
+	"github.com/go-chi/chi/v5"
+)
+
+func AuthRoute() chi.Router {
+	r := chi.NewRouter()
+	r.Get("/registerRequest", controller.RegisterRequest)
```

Comment:
This should be POST, not GET.
Right now it is using hard-coded info from the backend, but later it will need to identify who is registering, which will require the client to send data.

Reply:
stop stealing my comments

Reply:
:shipit:

Reply:
I think get is fine, it will send the token as a url param which is all the data we need. Otherwise, I would prefer not to POST since we're not doing any creation/deletion action. Let me know your thoughts.

Reply:
I will change it to POST